keyboards/helix/rev2/rev2.c

```diff
@@ -18,13 +18,30 @@ void led_set_kb(uint8_t usb_led) {
 }
 #endif
 
+bool is_mac_mode(void) {
+    // This is the opposite of the QMK standard, but we'll leave it for backwards compatibility.
+    return keymap_config.swap_lalt_lgui == false;
+}
+
+void set_mac_mode_kb(bool macmode) {
+    if (macmode) {
+        /* The result is the same as pressing the AG_NORM(=MAGIC_UNSWAP_ALT_GUI) key. */
+        /* https://github.com/qmk/qmk_firmware/blob/fb4a6ad30ea7a648acd59793ed4a30c3a8d8dc32/quantum/process_keycode/process_magic.c#L123-L124 */
+        keymap_config.swap_lalt_lgui = keymap_config.swap_ralt_rgui = false;
+    } else {
+        /* The result is the same as pressing the AG_SWAP(=MAGIC_SWAP_ALT_GUI) key. */
+        /* https://github.com/qmk/qmk_firmware/blob/fb4a6ad30ea7a648acd59793ed4a30c3a8d8dc32/quantum/process_keycode/process_magic.c#L80-L81 */
+        keymap_config.swap_lalt_lgui = keymap_config.swap_ralt_rgui = true;
+    }
+}
```

Comment:
Wouldn't this be simpler as: 
```suggestion
    /* The result is the same as pressing the AG_NORM_/AG_SWAP keys. */
    /* https://github.com/qmk/qmk_firmware/blob/fb4a6ad30ea7a648acd59793ed4a30c3a8d8dc32/quantum/process_keycode/process_magic.c#L123-L124 */
    keymap_config.swap_lalt_lgui = keymap_config.swap_ralt_rgui = !macmode;
    eeconfig_update_keymap(keymap_config.raw);

}
```

That way, it's one line, does the same thing, and added, writes to eeprom (so it's properly saved, as in process_magic: 
https://github.com/qmk/qmk_firmware/blob/56933b9b9f00a46980c1201a6ba5d0c5c9932275/quantum/process_keycode/process_magic.c#L169

Reply:
Done.
Thanks!